src/client/android.ts

```diff
@@ -46,7 +46,7 @@ export class Android extends ChannelOwner<channels.AndroidChannel, channels.Andr
 
   setDefaultTimeout(timeout: number) {
     this._timeoutSettings.setDefaultTimeout(timeout);
-    this._channel.setDefaultTimeoutNoReply({ timeout });
+    void this._channel.setDefaultTimeoutNoReply({ timeout });
```

Comment:
Let's do a catch for all NoReply methods?

Reply:
Done.

Reply:
Oops, only did it for the new ones, updating existing as well.

Reply:
Since these are protocol calls, added .catch blocks.